source/server/config/http/cors.cc

```diff
@@ -0,0 +1,31 @@
+#include "server/config/http/cors.h"
+
+#include <chrono>
+#include <cstdint>
+#include <string>
+
+#include "envoy/registry/registry.h"
+
+#include "common/http/filter/cors_filter.h"
+#include "common/json/config_schemas.h"
```

Comment:
nit: not needed (maybe others also, please clean up BUILD file also)

Reply:
Oh, sorry. I'll clean that up. Totally used to `go` cleaning up imports by itself.